templates/templates/footer.html

```diff
@@ -18,6 +18,9 @@
           <li class="p-inline-list__item">
             <a class="p-link--soft p-link--external" href="https://github.com/canonical-websites/cn.ubuntu.com/issues/new">Report a bug with this site</a>
           </li>
+          <li class="p-inline-list__item">
```

Comment:
same question as jp, should this go before the 'report a bug' link?

Reply:
Done.